src/main/webapp/i18n/de/aboutUs.json

```diff
@@ -16,20 +16,20 @@
             "title": "Artemis unterstützt folgende Features für Klausuren",
             "detail": {
                 "supportedExerciseTypes": "Alle Aufgabentypen sind auch in der Klausur verfügbar",
-                "randomSelectionOfVariants": "Mehrere Varianten von Aufgaben können erstellt und den Studenten zufällig zugewiesen werden",
+                "randomSelectionOfVariants": "Mehrere Varianten von Aufgaben können erstellt und den Studierenden zufällig zugewiesen werden",
                 "fullFeatureOverview": {
                     "description": "Gesamter Überblick über alle Features:",
-                    "studentLink": "Überblick für Studenten",
-                    "instructorLink": "Überblick für Dozenten"
+                    "studentLink": "Überblick für Studierende",
+                    "instructorLink": "Überblick für Lehrpersonen"
```

Comment:
I really like the changes overall, but I am not sure about the use of 'Lehrpersonen'. The problem is that this word does not exist. I googled quite a bit, its apparently only used a historical swiss-word. I think we should be strict with only using gender inclusive language, but not invent new words or use words that are just plain wrong in this context. 😃  If we want to be super strict, we have to use 'Dozenten und Dozentinnen' or 'Dozentenschaft' or 'Lehrende' or maybe someone else has an even better idea? 😄